packages/core/src/mechanic-validation.js

```diff
@@ -218,15 +225,21 @@ const prepareValues = (inputs, settings, baseValues) => {
  */
 const isSVG = el => el instanceof SVGElement;
 
+/**
+ * Checks whether a DOM element is instance of HTMLCanvasElement
+ * @param {object} el - A DOM element to check
+ */
+const isCanvas = el => el instanceof HTMLCanvasElement;
+
 /**
  * Validates that a DOM element is SVG or Canvas
  * @param {object} el - A DOM element to check
  */
 const validateEl = el => {
-  if (isSVG(el) || el instanceof HTMLCanvasElement) {
+  if (isSVG(el) || isCanvas(el) || el instanceof HTMLElement) {
```

Comment:
Just out of curiosity, aren't canvas elements also recognized as HTMLElement?

Reply:
True - have been updated in [19211bc](https://github.com/designsystemsinternational/mechanic/pull/113/commits/19211bce4ad9cd54e93590b69c124a9581ea4599).